docs/model_library/operational_water_management/index.rst

```diff
@@ -459,8 +460,9 @@ For each storage site and each time period, the volume of water being trucked in
 
 .. math::
 
-    \sum\nolimits_{(n,s)\in NSA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(p,s)\in PST}\textcolor{red}{F_{l,l,t}^{Trucked}} +\sum\nolimits_{(p,s)\in CST}\textcolor{red}{F_{l,l,t}^{Trucked}} \leq \textcolor{green}{\sigma_{s}^{Processing,Storage}}
-
+    \sum_{l \in L | (l, s) \in LLA}\textcolor{red}{F_{l,s,t}^{Piped}}
+        + \sum_{l \in L | (l, s) \in LLT}\textcolor{red}{F_{l,s,t}^{Trucked}}
+        \leq \textcolor{green}{\sigma_{s}^{Processing,Storage}}
 
 
 **Production Tank Balance:**
```

Comment:
Production Tank Balance documentation would benefit from being rewritten separately for the t = 1 case and the t > 1 case.

Reply:
Addressed.